include/nuSQuIDS/body.h

```diff
@@ -762,6 +766,123 @@ class EarthAtm: public Body{
     Track MakeTrackWithCosine(double cosphi);
 };
 
+
+
+/// \class EmittingEarthAtm
+/// \brief A model of the production of neutrino cosmic ray flux production within the atmosphere.
+class EmittingEarthAtm: public EarthAtm{
+  protected:
+    /// \brief Atm differential nu electron flux array
+    std::vector<double> diff_enu_prod;
+    marray<double,1> enu_flux;
+    /// \brief Atm differential nu muon  flux array
+    std::vector<double> diff_munu_prod;
+    marray<double,1> munu_flux;
+    /// \brief Atm coszen array
+    std::vector<double> atm_coszen;
+    marray<double,1> coszens;
+    /// \brief Atm heights array
+    std::vector<double> atm_heights;
+    marray<double,1> heights;
+    /// \brief Atm energies array
+    std::vector<double> atm_energy;
+    marray<double,1> energies;
```

Comment:
`EmittingEarthAtm` seems to contain a large number of arrays of duplicate data, e.g. `atm_coszen` and `coszens`. At a minimum we should not keep these duplicates, and it isn't clear why most of these things need to be member variables; many look like they could just be local variables within the constructor which takes a file path.